docs/content/getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+Navigate to your list of apps and [create a new Slack app](https://api.slack.com/apps/new) using the "from a manifest" option:
+
+1. Select the workspace to develop your app in.
+2. Use copied values from the `manifest.json` file to create your app.
+3. Confirm the app features and click "Create".
+
+You'll then land on your app's **Basic Information** page, which is an overview of your app and which contains important credentials:
+
+![Basic Information page](/img//basic-information-page.png "Basic Information page")
+
+To listen for events happening in Slack (such as a new posted message) without opening a port or exposing an endpoint, we will use [Socket Mode](/concepts/socket-mode). This connection requires a specific app token:
+
+1. On the **Basic Information** page, scroll to the "App-Level Tokens" section and click **Generate Token and Scopes**.
+2. Name the token "Development" or something similar and add the `connections:write` scope, then click "Generate".
+3. Save the generated `xapp` token as an environment variable within your project:
+
+```sh
+$ export SLACK_APP_TOKEN=xapp-1-A0123456789-example
+```
+
+The above command works on Linux and macOS but [similar commands are available on Windows](https://superuser.com/questions/212150/how-to-set-env-variable-in-windows-cmd-line/212153#212153).
+
+:::warning[Keep it secret. Keep it safe]
+
+Treat your tokens like a password and [keep it safe](https://docs.slack.dev/authentication/best-practices-for-security). Your app uses these to retrieve and send information to Slack.
+
+:::
+
+A bot token is also needed to interact with the Web API methods as your app's bot user. We can gather this as follows:
+
+1. Navigate to the **OAuth & Permissions** on the left sidebar and install your app to your workspace to generate a token.
+2. After authorizing the installation, you'll return to the **OAuth & Permissions** page and find a **Bot User OAuth Access Token**:
+
+![OAuth Tokens](/img//bot-token.png "Bot OAuth Token")
+
+3. Copy the bot token beginning with `xoxb` from the **OAuth & Permissions page** and then store it in a new environment variable:
+
+```sh
+$ export SLACK_BOT_TOKEN=xoxb-example
+```
+
+After the app has been created, it is time to run it:
+
+```sh
+$ npm run start
+...
+[INFO]  bolt-app ⚡️ Bolt app is running!
+```
+
+With the app running, you can test it out with the following steps:
+
+1. Open a direct message with your app or invite the bot to a public channel.
+2. Send "hello" to the current conversation and wait for a response.
+3. Click the attached button labelled "Click me" to post another reply.
+
+After confirming the app responds, celebrate, then interrupt the process by pressing `CTRL+C` in the terminal to stop your app from running.
+
+</TabItem>
+</Tabs>
+
+## Updating the app
+
+At this point, you're successfully running the getting started Bolt for JavaScript [app](https://github.com/slack-samples/bolt-js-getting-started-app)!
```

Comment:
suggestion: In the previous section, we concluded with `CTRL+C` so the app isn't currently running.
```suggestion
At this point, you've successfully run the getting started Bolt for JavaScript [app](https://github.com/slack-samples/bolt-js-getting-started-app)!
```